torch/onnx/utils.py

```diff
@@ -401,11 +405,34 @@ def _get_param_count_list(method_graph, args_params):
         if "PackedParams" in str(input_.type()):
             in_vars, _ = torch.jit._flatten(arg_params_)
             param_count_list.append(len(in_vars))
+        elif arg_params_ is None:
+            param_count_list.append(0)
         else:
             param_count_list.append(1)
     return param_count_list
 
 
+def _resolve_and_flatten_graph_inputs(model, args_params):
+    sig = _signature(model)
+    param_keys = list(sig.parameters.keys())
+    resolved_args = []  # type: ignore[var-annotated]
+    if isinstance(model, (torch.jit.ScriptFunction, torch.jit.ScriptModule)):
+        for i, var in enumerate(args_params):
+            if var is None:
```

Comment:
I feel the purpose is to grab default value when argument `var` is omitted in example inputs. But how does it work? I thought `var is None` is supposed to mean `None` passed as input explicitly. Let me know where I'm wrong.